addons/calendar/controllers/main.py

```diff
@@ -60,18 +60,12 @@ def view(self, db, token, action, id, view='calendar'):
     # Function used, in RPC to check every 5 minutes, if notification to do for an event or not
     @http.route('/calendar/notify', type='json', auth="none")
     def notify(self):
-        registry = request.registry
-        uid = request.session.uid
-        context = request.session.context
-        with registry.cursor() as cr:
-            res = registry.get("calendar.alarm_manager").get_next_notif(cr, uid, context=context)
-            return res
+        with request.env.registry.cursor() as cr:
+            env = request.env(cr, request.session.uid, context=request.session.context)
+            return env["calendar.alarm_manager"].get_next_notif()
```

Comment:
what if I simply write ``` env = request.env(user=request.session.uid, context=request.session.context) ```
instead of line 63 and 64, is it required to take new cursor here ?